.github/workflows/canary-integration-test.yml

```diff
@@ -238,12 +238,12 @@ jobs:
           name: canary
 
       - name: consider debugging
-        if: failure() && github.event_name == 'pull_request'
-        timeout-minutes: 60
+        if: always()
```

Comment:
will revert this after someone else verifies and approval.

Reply:
https://github.com/rook/rook/actions/runs/5485094447/jobs/9993477187?pr=12484#step:28:50

```
SSH: ssh KEJWAXrKKgPgVhHT3HAWrETFz@sfo2.tmate.io
Web shell: https://tmate.io/t/KEJWAXrKKgPgVhHT3HAWrETFz
SSH: ssh KEJWAXrKKgPgVhHT3HAWrETFz@sfo2.tmate.io```

Reply:
ig the above check would be fine

Reply:
yes, I will revert once someone else verify those acces